core/shared/mem-alloc/ems/ems_gc_internal.h

```diff
@@ -310,7 +310,8 @@ typedef struct gc_heap_struct {
     gc_size_t gc_threshold;
     gc_size_t gc_threshold_factor;
     gc_size_t total_gc_count;
-    gc_size_t total_gc_time;
+    double total_gc_time;
+    double max_gc_time;
```

Comment:
How about using uint32 or uint64, and using os_time_get_boot_microsecond to get the time?